packages/twenty-front/src/modules/object-record/record-show/components/FieldsCard.tsx

```diff
@@ -26,13 +26,15 @@ import { FieldMetadataType } from '~/generated-metadata/graphql';
 type FieldsCardProps = {
   objectNameSingular: string;
   objectRecordId: string;
+  isMergeMode?: boolean;
 };
 
 const INPUT_ID_PREFIX = 'fields-card';
 
 export const FieldsCard = ({
   objectNameSingular,
   objectRecordId,
+  isMergeMode = false,
```

Comment:
It's better to introduce a more re-usable property that describes the goal not the context so rename it to something like showDuplicatesSection